docs/source/examples/ddd_on_ibmq_ghz.md

```diff
@@ -11,236 +11,253 @@ kernelspec:
+        print("Result:", noisy_value)
+        data.append((depth, rule, noisy_value))
 ```
 
 Now we can visualize the results.
 
 ```{code-cell} ipython3
-avg_true_values = np.average(true_values, axis=1)
-avg_noisy_values = np.average(noisy_values, axis=1)
-
-std_true_values = np.std(true_values, axis=1, ddof=1)
-std_noisy_values = np.std(noisy_values, axis=1, ddof=1)
-
-avg_ddd_values = np.average(ddd_values, axis=1)
-std_ddd_values = np.std(ddd_values, axis=1, ddof=1)
-
-plt.figure(figsize=(9, 5))
-
-plt.plot(num_qubits, avg_true_values, "--", label="True", lw=2)
-eb = plt.errorbar(
-    num_qubits, avg_noisy_values, yerr=std_noisy_values, label="Raw", ls="-."
-)
-eb[-1][0].set_linestyle("-.")
-plt.errorbar(num_qubits, avg_ddd_values, yerr=std_ddd_values, label="DDD")
-
-plt.title(
-    f"""Simulator with GHZ circuits using DDD {num_qubits} \nqubits, {trials} trials."""
-)
-plt.xlabel("Number of Qubits")
-plt.ylabel("Expectation value")
-_ = plt.legend()
+:tags: ["remove-output"]
+# Plot unmitigated
+x, y = [], []
+for res in data:
+    if res[1].__name__ == "rep_i_rule":
+        x.append(res[0])
+        y.append(res[2])
+plt.plot(x, y, "--*", label="Unmitigated")
+
+# Plot xx
+x, y = [], []
+for res in data:
+    if res[1].__name__ == "rep_xx_rule":
+        x.append(res[0])
+        y.append(res[2])
+plt.plot(x, y, "--*", label="rep_xx_rule")
+
+# Plot ixix
+x, y = [], []
+for res in data:
+    if res[1].__name__ == "rep_ixix_rule":
+        x.append(res[0])
+        y.append(res[2])
+plt.plot(x, y, "--*", label="rep_ixix_rule")
+
+# Plot xx
+x, y = [], []
+for res in data:
+    if res[1].__name__ == "xx":
+        x.append(res[0])
+        y.append(res[2])
+plt.plot(x, y, "--*", label="xx")
+
+
+plt.legend()
 ```
 
-We can see that on average DDD improves the expectation value at each circuit width. 
-Note that the size of the error bars represents the standard deviation of the noisy values (for the “Raw” line) and the standard deviation
-of the DDD values (for the “DDD” line).
-The improvement increases with circuit size, which is expected given the strongly time-correlated dephasing noise applied in this example.
-In general, real hardware would exhibit a different noise model from what is shown here, but real devices usually have some time-correlated noise
-that can be mitigated by dynamical decoupling.
+```{figure} ../_thumbnails/ddd_qiskit_ghz_plot.png
+---
+
+name: ddd-qiskit-ghz-plot-ibmq
+---
+Plot of the unmitigated and DDD-mitigated expectation values obtained from executing the corresponding circuits.
+```
+
+
+We can see that DDD improves the expectation value at each circuit depth, and the repeated XX sequence is the best at mitigating the errors
+occurring during idle windows. 
+This is expected because the idle windows are long compared to the length of the other operations in the circuit, so repeated pulses close
+together in time would more effectively decouple the qubits from the environment.
```

Comment:
I don't feel we have enough certainty on the reason why XX is doing better. According to your comment, repeated XX rule should **always** be better than non-repeated sequences. But don't think this is always true in practical experiments.

I propose we remove or we put it in a more speculative way.  E.g. something like: "A possible reason why repeated xx is more effective is ...." 


```suggestion

```